src/unix/bsd/freebsdlike/freebsd/freebsd11/mod.rs

```diff
@@ -424,6 +424,17 @@ extern "C" {
     // in FreeBSD 12
     pub fn dirname(path: *const c_char) -> *mut c_char;
     pub fn basename(path: *const c_char) -> *mut c_char;
+
+    // Argument order of the function pointer changed in FreeBSD 14. From 14 onwards the signature
+    // matches the POSIX specification by having the third argument be a mutable pointer, on
+    // earlier versions the first argument is the mutable pointer.
+    pub fn qsort_r(
```

Comment:
```suggestion
    #[link_name = "qsort_r@FBSD_1.0"]
    pub fn qsort_r(
```

Reply:
I think you need to specify the link_name here.  Otherwise, you will link to the wrong symbol if you build CI in FreeBSD 11 compatible mode (or 12, or 13) but run if on FreeBSD 14 or 15.  For example, if you try to run `sh ci/run.sh x86_64-unknown-freebsd` on FreeBSD 14 or 15 (note the lack of `LIBC_CI=1`).